modules/administration/pages/ansible-integration.adoc

```diff
@@ -0,0 +1,52 @@
+[[ansible-integration]]
+= Ansible integration
+
+[NOTE]
+====
+This feature is a techhnological preview.
+====
+
+[NOTE]
+====
+Currently, the supported version of Ansible is 2.9 (LTS) shipped in SUSE Linux Enterprise 15 Client Tools channels. For Control Nodes running operating systems other than SUSE Linux Enterprise, use Ansible shipped by your distribution.
+====
+
+[[at.ansible.overview]]
+== Feature Overview
+
+{productname} enables system administrators to operate their Ansible Control Nodes. Supported features are:
+* introspection of inventory files
+* discovery of playbooks
+* execution of playbooks
+
+
+[[at.ansible.requirements]]
+== Requirements
+
+To use Ansible features, you need to register your Ansible Control Node system as a Salt client in your {productname}. Moreover, you must enable the ``Ansible Control Node`` system type in the [guimenu]``Add-on System Types`` in the menu:System Details[Properties] page of that system.
+
+Enabling this system type ensures that the ``ansible`` package is installed on the system by adding it in the highstate and enables the Ansible features under the menu::System[Ansible] page.
+
+As the next step, configure the paths to your Ansible inventories and playbooks in the menu::System[Ansible][Control Node]. As an invertory path, you can use the typical ansible inventory path [literal]``/etc/ansible/hosts``. As a playbook directory, you can use any directory on the control node, where your playbooks reside (typically directories containing the [literal]``site.yml`` files).
```

Comment:
A note here about the "playbook directories".

The current text makes the impression that you have to introduce the playbook directory where your yml file is, but that's not the only case since the discovery mechanism also explores the 1st level of subdirectories looking for a `site.yml`. So, you can introduce a path, like `/srv/playbooks` that contains subdirectories with playbooks on it.

All those playbook would be exposed.

Reply:
That's a good point, I tried to re-phase it. Please check if it makes sense now.